Detections/AzureDiagnostics/TimeSeriesKeyvaultAccessAnomaly.yaml

```diff
@@ -41,9 +44,16 @@ query: |
   | join (
   AzureDiagnostics
   | where TimeGenerated between (startofday(ago(starttime))..startofday(now()))
+  | extend ResultType = columnifexists("ResultType", "None"), identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g = columnifexists("identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g", "None")
+  | extend id_s = columnifexists("id_s", "None"), CallerIPAddress = columnifexists("CallerIPAddress", "None"), clientInfo_s = columnifexists("clientInfo_s", "None")
+  | where ResultType != "None" and isnotempty(ResultType)
+  | where identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g != "None" and isnotempty(identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g)
+  | where id_s != "None" and isnotempty(id_s)
+  | where CallerIPAddress != "None" and isnotempty(CallerIPAddress)
+  | where clientInfo_s != "None" and isnotempty(clientInfo_s)
   | where ResourceType == "VAULTS" and ResultType == "Success"
   | where OperationName in (OperationList)  
   | summarize PerOperationCount=count() by bin(TimeGenerated,1h), Resource, OperationName, id_s, CallerIPAddress, identity_claim_http_schemas_microsoft_com_identity_claims_objectidentifier_g, requestUri_s, clientInfo_s     
```

Comment:
requestUri_s is not a property that is defined.

Reply:
added fix.